pkgs/top-level/perl-packages.nix

```diff
@@ -2327,6 +2327,37 @@ let
      };
   };
 
+  ClusterSSH = buildPerlPackage rec {
+    name = "App-ClusterSSH-4.13.2";
+    src = fetchurl {
+      url = mirror://cpan/authors/id/D/DU/DUNCS/App-ClusterSSH-v4.13.2.tar.gz;
+      sha256 = "0rmk2p3f2wz1h092anidjclh212rv3gxyk0c641qk3frlrjnw6mp";
+    };
+    propagatedBuildInputs = [ ExceptionClass FilePath GetoptLong LocaleMaketext Tk TryTiny X11Protocol X11ProtocolOther pkgs.coreutils pkgs.openssh pkgs.xterm ];
+    buildInputs = [ FileSlurp FileTemp FileWhich CPANChanges TestPerlTidy TestPod TestPodCoverage Readonly TestDifferences TestDistManifest TestTrap ];
+    preConfigure = ''
+      for fn in bin_PL/* ; do
+        substituteInPlace "$fn" --replace '/usr/bin/env perl' '/usr/bin/perl'
+      done
```

Comment:
the `patchShebangs` function should fix this also

Reply:
Ok. Wait a sec, why are you replacing it to `/usr/bin/perl`?

Reply:
This is what happens without it:
```
$ cssh
/nix/store/dzvbhkzfgwkq97ljkcvi4gs7vmflh6a0-coreutils-8.30/bin/env: ‘perl -I/nix/store/0ingn8cwwnl84i374hcl6nafsm2c5m2p-perl-5.’: No such file or directory
/nix/store/dzvbhkzfgwkq97ljkcvi4gs7vmflh6a0-coreutils-8.30/bin/env: use -[v]S to pass options in shebang lines
```
As I understand it, Linux allows only one argument to shebang, so `env` takes `perl` and all its path arguments as a single argument and tries to execute it. Patching to `/usr/bin/perl` solves this, I guess perl handles this situation. I can change it if you have a better idea.